### PR TITLE
java 17 & boot

### PR DESCRIPTION
**Pull request by kylehoehns**

### CODE REVIEW

File: .devcontainer.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
 	"name": "Mono",
 	"image": "mcr.microsoft.com/devcontainers/universal:focal",
 	"postAttachCommand": "source ./init.sh",
+	"containerEnv": {
```
Comment (kylehoehns) [inline .devcontainer.json:5]: This is the magic that defaults us to using 17 instead of 11 🤦

**Review by nbatterson — APPROVED** [review]

File: .devcontainer.json (hunk 1 above)
Comment (tim-hornback-SAI) [reply .devcontainer.json:5]: in what world would I ever think that DISABLE_JAVA_BUILD => Java 17 wild

File: java-spring-boot/src/.DS_Store
Comment (tim-hornback-SAI) [inline java-spring-boot/src/.DS_Store:1]: what is this guy doin here

File: java-spring-boot/build.gradle
Comment (tim-hornback-SAI) [inline java-spring-boot/build.gradle:1]: I can't believe you've disgraced Spring with Gradle

**Review by tim-hornback-SAI — APPROVED** [review]

File: java-spring-boot/build.gradle
Comment (kylehoehns) [reply java-spring-boot/build.gradle:1]: It's the default now! :D

**Review by tim-hornback-SAI — APPROVED** [review]

**Pull request merged**